### PR TITLE
Announcements schema

### PR DESCRIPTION
**Pull request by AndrewBoland**
Fixes # .
#105 

### PR Description
Implements schema for announcements

### What needs to be reviewed?
server/db/model/Announcement.js

### Context?
I wasn't sure which queries to implement so I added a few, feedback as to which to take out, modify or add would be great.

### CODE REVIEW

File: server/db/model/Announcement.js [hunk 1]
```diff
@@ -2,3 +2,72 @@
...
+});
+
+// Allow us to query by title
+schema.query.byTitle = function(title) {
+    return this.findOne({
+        title: new RegExp(title, 'i')
+    });
+};
+
+// Allow us to query by token
```
Comment (connorkrupp) [inline server/db/model/Announcement.js]: Update this comment you silly goose

File: server/db/model/Announcement.js [hunk 2]
```diff
@@ -2,3 +2,72 @@
...
+
+// Allow us to query by isSent
+schema.query.byIsSent = function() {
+    return this.find({
+        isSent: true
+    });
+};
+
+// Initialize the model with the schema, and export it
+var model = mongoose.model('Announcements', schema);
```
Comment (connorkrupp) [inline server/db/model/Announcement.js]: Make 'Announcements' singular

File: server/db/model/Announcement.js [hunk 3]
```diff
@@ -2,3 +2,72 @@
...
+    });
+};
+
+// Allow us to query by isSent
+schema.query.byIsSent = function() {
+    return this.find({
+        isSent: true
+    });
+};
+
```
Comment (connorkrupp) [inline server/db/model/Announcement.js]: Let's make a query for ones that are both approved and sent and call it `isPublic` or something better if you think of a better way to express it

**Comment by AndrewBoland** [comment]
I updated the byIsPublic query to query for time before now. I also fixed all the comments. Let me know if there is anything else I should do or if I should rename anything.

**Pull request merged**